docs/UserGuide.md

```diff
@@ -241,8 +242,10 @@ Parameters:
 Restrictions:
 * All parameters must be separated by a single space.
 * All arguments cannot be blank.
+* Events cannot have duplicate names(Event names are unique to each event).
 * The date and time formats must be exactly `DD/MM/YYYY TTTT`.
 * If the end date and time is specified, it must be the _same time_ or _after_ the start date and time of the event.
+* The location and description arguments cannot contain the `[` and `]` symbols.
```

Comment:
oo is this a limitation of the regex engine? cos i think can use `\[` and `\]` to bypass this issue

Reply:
ya i think cannot leh. my current regex alr has `\` before the `]`

Reply:
try the test case `ecreate n/go jb shopping sd/31/12/2023 2359 r/10 shopper l/j]b m/20 shopping bags dsc/shopping]]]`

Reply:
it's possible that `]` works while `[` doesn't cos i don't see `\[` in the regex

Reply:
wah lets go jb ah

Reply:
yessir! anyway does it work lol